packages/apollo-server-core/src/runHttpQuery.ts

```diff
@@ -114,7 +114,8 @@ export async function runHttpQuery(
   handlerArguments: Array<any>,
   request: HttpQueryRequest,
 ): Promise<HttpQueryResponse> {
-  function debugFromNodeEnv(nodeEnv: string | undefined) {
+  function debugFromNodeEnv() {
+    const nodeEnv = request.options.nodeEnv;
```

Comment:
I was getting a linting error here that said `nodeEnv` wasn't on the type. But the type is `GraphQLServerOptions`, which does have `nodeEnv`.

Reply:
Also saw no reason to pass in `nodeEnv` when it would already be available in scope here.